### PR TITLE
Added semicolons in duckdb.mdx 

### PR DESCRIPTION
**Pull request by anubhav1206**
## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #(issue)

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@anubhav1206 
Please follow the rules as outlined below:

1. Leave a comment on an issue you want to work on.
2. Wait until we assign you the issue.
3. Once you are assigned, you can create a PR.

To have your PR merged please sign our CLA by leaving a comment here that contains this line: I have read the CLA Document and I hereby sign the CLA

File: docs/data-integrations/duckdb.mdx [hunk 1]
```diff
@@ -28,7 +28,7 @@ The optional arguments are as follows:
 In order to make use of this handler and connect to the DuckDB database in MindsDB, the following syntax can be used:
 
 ```sql
-CREATE DATABASE duckdb_datasource
+CREATE DATABASE duckdb_datasource;
```
Comment (martyna-mindsdb) [inline docs/data-integrations/duckdb.mdx:31]: The semicolon should be at the end of the statement and not in its first line. Please correct it.

File: docs/data-integrations/duckdb.mdx [hunk 2]
```diff
@@ -39,6 +39,6 @@ WITH
 You can use this established connection to query your table as follows:
 
 ```sql
-SELECT *
+SELECT *;
```
Comment (martyna-mindsdb) [inline docs/data-integrations/duckdb.mdx:42]: The semicolon should be at the end of the statement and not in its first line. Please correct it.

**Comment by martyna-mindsdb** [comment]
@anubhav1206 
Please always link the issue number in PR description, like this: https://github.com/mindsdb/mindsdb/issues/5797

Also, please see and implement the review comments above.

**Comment by martyna-mindsdb** [comment]
Opened a new PR: https://github.com/mindsdb/mindsdb/pull/5815

**Pull request closed without merging**